plugins/dra-ocp-validator/tools/run-tests.sh

```
@@ -0,0 +1,220 @@
+if [ "${K8S_MINOR}" -ge 35 ]; then
+  AVAILABLE_FEATURES+=("extended-resources")
+fi
+
+# Parse features to test
+TEST_FEATURES=()
+if [ "${FEATURES}" = "all" ]; then
+  TEST_FEATURES=("${AVAILABLE_FEATURES[@]}")
+else
+  IFS=',' read -ra REQUESTED_FEATURES <<< "${FEATURES}"
+  for feature in "${REQUESTED_FEATURES[@]}"; do
+    if [[ " ${AVAILABLE_FEATURES[@]} " =~ " ${feature} " ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)